src/integration-test/tests/legal/home/pages/login.ts

```diff
@@ -0,0 +1,24 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  username: '#username',
+  password: '#password'
+}
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class LoginPage {
+  open () {
+    I.amOnLegalAppPage('/')
+  }
+
+  login (email, password) {
```

Comment:
Please add type definitions to method args and return value.

Reply:
done